.github/workflows/gh-pages.yml

```diff
@@ -0,0 +1,35 @@
+name: github pages
+
+on:
+  push:
+    branches:
+      - master  # Set a branch to deploy
+    paths:
+      - 'docs/**'
+  pull_request:
+
+jobs:
+  deploy:
+    runs-on: ubuntu-18.04
```

Comment:
can we use 20.04? Will last us longer?

Reply:
Sure! I'm a bit new to github actions, so this was largely copy/pasted, and this was just a byproduct of that.